src/main/test/surfrdUtils_test/test_surfrdUtils.pf

```diff
@@ -43,6 +43,104 @@ contains
 
   end subroutine setup_test_collapse_crop_types
 
+  @Test
```

Comment:
@ekluzek , it took me a little while to get this unit test working (for various reasons, including that I needed to use "tolerance" for the assert statements to pass). Now that it works, I'm open as always to comments and suggestions, if there are ways to make more readable etc...

Reply:
OK, I went through this and now understand it. I have a couple suggestions. Right now it puts in 4 active PFT's, and it has two extra ones that it sets to zero. It's only testing that you can go from the 4 down to 2, as well as the default cases of when you aren't collapsing.

Since, you already have the rows for pfts 2 and 3, you might as well put values inside it. They just need to be smaller than 30. Right now the first and second grid cell is the same for the PFT's that are being removed. You might as well make them different and maybe more random, since the 40, and 30 for pfts 0 and 5 is the only thing that's active. Having random values in there, just shows that they don't come into play in what happens (only the 40 and 30 for pfts 0 and 5).

You could also test more cases and also collapse to one (rather than just two). Technically you could show it for any valid number of n_dom_pfts, but I think that adding one more case would be sufficient.

Reply:
This is complete.